.travis.yml

```diff
@@ -14,12 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+language: ruby
 sudo: false
+cache: bundler
 
 before_install:
-  - gem update --system
-  - gem install bundler --no-ri --no-rdoc
-  - gem --version
+  - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
+  - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+  - rm -f .bundle/config
+
```

Comment:
Can you leave the gem --version call here?  Maybe even add a bundle --version?  Just to make debugging easier.